src/main/java/edu/greenblitz/robotName/utils/shootingCalculations/ShootingZone.java

```diff
@@ -0,0 +1,53 @@
+package edu.greenblitz.robotName.utils.shootingCalculations;
+
+
+import edu.greenblitz.robotName.shootingStateService.ShootingState;
+import edu.greenblitz.robotName.utils.GBMath;
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShootingZone extends GBMath.GBCircle {
+
+    private List<Pair<Translation2d, Translation2d>> restrictedBounds; //first is the lower bound, second is the upper
```

Comment:
you can maybe remove the comment and in the build function do the finding min max🛠️